LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2018, US Army Engineer Research and Development Center
```

Comment:
Should probably use the same language as erdc/proteus i.e.:

"Copyright (c) 2017 U.S. Federal Government (in countries where recognized)"